docs/output.md

```diff
@@ -377,7 +377,7 @@ The named sample, which just contains the sample name in the name is the final g
 
 Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQC. The pipeline has special steps which also allow the software versions to be reported in the MultiQC output for future traceability. For more information about how to use MultiQC reports, see <http://multiqc.info>.
 
-## Pipeline information
+### Pipeline information
```

Comment:
Why did you make it a subhearder of MultiQC?

Reply:
I would not change this deliberately. Maybe a Template thing, let me fix.